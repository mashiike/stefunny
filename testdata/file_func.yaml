required_version: ">v0.0.0"

state_machine:
  name: Hello
  definition: file_func.asl.json
  role_arn: arn:aws:iam::012345678901:role/service-role/StepFunctions-Hello-role
  logging_configuration:
    level: ALL
    destinations:
      - cloudwatch_logs_log_group:
          log_group_arn: arn:aws:logs:us-east-1:012345678901:log-group:/steps/hello
