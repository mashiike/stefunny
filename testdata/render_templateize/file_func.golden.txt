## config
{
  aws_region: 'us-east-1',
  required_version: '>v0.0.0',
  state_machine: {
    definition: 'file_func.asl.json',
    logging_configuration: {
      destinations: [
        {
          cloudwatch_logs_log_group: {
            log_group_arn: 'arn:aws:logs:us-east-1:012345678901:log-group:/steps/hello',
          },
        },
      ],
      level: 'ALL',
    },
    name: 'Hello',
    role_arn: 'arn:aws:iam::012345678901:role/service-role/StepFunctions-Hello-role',
    tracing_configuration: {},
    type: 'STANDARD',
  },
}
## definition
{
  Comment: '{{ template_file `comment.txt` | trim | json_escape }}',
  StartAt: 'Hello',
  States: {
    Hello: {
      Next: 'World',
      Type: 'Pass',
    },
    World: {
      End: true,
      Result: 'World',
      Type: 'Pass',
    },
  },
}
